client/my-sites/importer/newsletter/importer.tsx

```diff
@@ -1,8 +1,10 @@
+import { Spinner } from '@wordpress/components';
+import { Icon, check } from '@wordpress/icons';
 import { addQueryArgs, getQueryArg } from '@wordpress/url';
 import { useState, useEffect } from 'react';
 import { UrlData } from 'calypso/blocks/import/types';
 import FormattedHeader from 'calypso/components/formatted-header';
-import StepProgress from 'calypso/components/step-progress';
+import StepProgress, { ClickHandler } from 'calypso/components/step-progress';
```

Comment:
not sure if there's a rule but other imports dont use defaults when importing multiple

Reply:
The editor was complaining when I didn't use the default.